src/models/tiles/geometry/jsxgraph.d.ts

```diff
@@ -178,6 +178,7 @@ declare namespace JXG {
 
     findPoint: (point: JXG.Point) => number;
     removePoints: (...points: JXG.Point[]) => void;
+    addPoints: (...points: JXG.Point[]) => void;
```

Comment:
```suggestion
    addPoints: (...points: JXG.Point[]) => void;
    removePoints: (...points: JXG.Point[]) => void;
```
`add` before `remove` would be the more conventional order.